app/controllers/api/v1/budgets_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Api::V1::BudgetsController < ApplicationController
+  before_action :set_budget, only: [:update, :show]
```

Comment:
set_budget for the destroy method is needed too

Reply:
Hi enma. As you can see in the test the issue is not complete, I just made the PR because initially the issue was about index only, but last night I decided to chaged it to the complete CRUD XD. When I finish the issue I will add you as a reviewer. :)